gallery/src/components/NxAccordion/NxAccordionPage.tsx

```diff
@@ -4,184 +4,265 @@
-            </NxTable.Row>
-            <NxTable.Row>
-              <NxTable.Cell><NxCode>nx-h3</NxCode></NxTable.Cell>
-              <NxTable.Cell>Subheader within accordion body</NxTable.Cell>
-              <NxTable.Cell>
-                The contents of the accordion body may include subheaders which should
-                be <NxCode>&lt;h3&gt;</NxCode> elements with
-                the <NxCode>.nx-h3</NxCode> class.
-              </NxTable.Cell>
-            </NxTable.Row>
-          </NxTable.Body>
-        </NxTable>
-      </NxTile.Subsection>
-    </GalleryDescriptionTile>
+              Note that this component is stateless – its open state must be tracked externally.
+              See <NxCode>NxStatefulAccordion</NxCode> for a version which tracks its own open state.
+            </NxP>
+            <NxH3>Stateful Accordion</NxH3>
+            <NxP>
+              <NxCode>NxStatefulAccordion</NxCode> is a wrapper
+              around <NxCode>NxAccordion</NxCode> which tracks its own toggle state. It accepts
+              the same props as <NxCode>NxAccordion</NxCode>, except that instead
+              of <NxCode>open</NxCode>, it accepts <NxCode>defaultOpen</NxCode> which
+              provides the initial toggle state.
+            </NxP>
+            <NxH2>Guidelines</NxH2>
+            <NxH3>When to Use</NxH3>
+            <NxList bulleted>
+              <NxList.Item>Displaying and grouping additional information.</NxList.Item>
+              <NxList.Item>Adding granular control over the information on a given page.</NxList.Item>
+              <NxList.Item>Shortening pages to reduce scrolling.</NxList.Item>
+            </NxList>
+            <NxH3>When Not to Use</NxH3>
+            <NxList bulleted>
+              <NxList.Item>
+                Displaying critical system information or a primary action to be taken on the page. (for example,
+                {' '}<a href="/components/alert">alerts</a>, confirmation or cancellation buttons).
+              </NxList.Item>
+              <NxList.Item>
+                Displaying navigation elements such as <a href="/components/tabs">tabs</a>.
+              </NxList.Item>
+              <NxList.Item>
+                Displaying links pointing to sections of the same page, instead use a
+                {' '}<a href="/components/list">list</a>.
+              </NxList.Item>
+              <NxList.Item>
+                Creating hierarchy levels by nesting them within each other. If you need to add hierarchy to the
+                content use a <a href="/components/tree">tree</a>.
+              </NxList.Item>
+              <NxList.Item>
+                Displaying a set of visual components following the same style, prefer using Collapsable.(TODO: Add
+                link to collapsable).
+              </NxList.Item>
+            </NxList>
+            <NxTile.Subsection>
+              <NxTile.SubsectionHeader>
+                <NxH3>NxAccordion</NxH3>
```

Comment:
Why are the subheaders above this point not also in tile subsections?

Reply:
> Why are the subheaders above this point not also in tile subsections?

https://github.com/sonatype/sonatype-react-shared-components/pull/607/commits/cc91e7c1fa1737f501917c78fa45f8119d43f9d7